system/liquid-ceph/ci/test-values.yaml

```diff
@@ -6,7 +6,11 @@ global:
   linkerd_requested: false
 ceph:
   secretName: ceph-secret
-  endpoint: "http://ceph-rgw.ceph.svc.cluster.local:80"
-  targetZone: the-zone
-  validPools: the-pools
-  rgwSecretName: rgw-secret
+  zoneMapping:
+    az-a-3:
```

Comment:
can we reuse the `rgwTargetPlacements` and `premiumPlacements` mappings from the system/cc-ceph chart? duplicating zone mappings adds flexibility, but also a complexity and a room for a human error.

Reply:
cc @sumitarora2786